app/src/main/java/com/celzero/bravedns/util/OrbotHelper.kt

```diff
@@ -91,6 +93,8 @@ class OrbotHelper(private val persistentState: PersistentState, private val prox
         private const val EXTRA_HTTP_PROXY_PORT = "org.torproject.android.intent.extra.HTTP_PROXY_PORT"
 
         private const val EXTRA_STATUS = "org.torproject.android.intent.extra.STATUS"
+
+        private const val ORBOT_REQUEST_CODE = 200 //Orbot App
```

Comment:
`// Orbot app` code comment isn't adding anything of note here.

Reply:
Modified the comment.